bigchaindb/events.py

```diff
@@ -0,0 +1,33 @@
+from enum import Enum
+from multiprocessing import Queue
+
+
+class EventTypes(Enum):
+    BLOCK_VALID = 1
+    BLOCK_INVALID = 2
+
+
+class Event:
+
+    def __init__(self, event_type, event_data):
+        self.type = event_type
+        self.data = event_data
+
+
+class EventHandler:
+
+    def __init__(self, events_queue):
+        self.events_queue = events_queue
+
+    def put_event(self, event, timeout=None):
+        # TODO: handle timeouts
```

Comment:
@libscott: hybrid approach, it's fine to have `TODO`s as long as they reference an issue in our tracker. What do you think?

@r-marques: can you write an issue for this `TODO` please?